x-pack/legacy/plugins/siem/public/components/open_timeline/types.ts

```diff
@@ -36,6 +36,10 @@ export interface TimelineActionsOverflowColumns {
   } | null>;
 }
 
+enum TimelineTypes {
```

Comment:
can we add this enum in io-ts in the common folder so we do not have to repeated